server/src/main/java/io/spine/server/tuple/package-info.java

```diff
@@ -19,42 +19,57 @@
  */
 
 /**
- * This package provides tuples for return values from
- * {@linkplain io.spine.server.command.Assign command handling} or
- * {@linkplain io.spine.server.command.Command commanding} methods.
+ * This package provides classes for return values from message handling methods.
```

Comment:
I would say `... from the message-handling methods.` It's easier to comprehend, as it becomes an adjective and a noun instead of a noun-adjective-noun.